libr/core/cmd_debug.c

```diff
@@ -4725,11 +4725,25 @@ static int cmd_debug_step (RCore *core, const char *input) {
 	return 1;
 }
 
-static ut8*getFileData(RCore *core, const char *arg) {
+static ut8 *getFileData(RCore *core, const char *arg, int *sz) {
+	ut8 *out;
+	int size;
 	if (*arg == '$') {
-		return (ut8*) r_cmd_alias_get (core->rcmd, arg, 1);
+		RCmdAliasVal *v  = r_cmd_alias_get (core->rcmd, arg);
+		out = malloc (v->sz);
```

Comment:
Check for null

Reply:
also missed doing arg+1 here in alias_get

Reply:
(aliases arent stored with $ anymore)